.travis.yml

```diff
@@ -12,6 +12,7 @@ install:
   - bower install
 before_script:
   - gulp hooks:lint
+  - npm run karma-test
```

Comment:
currently just running the karma tests before the main suite, perhaps later we can find a way to run it all as part of `gulp test`